src/dev/DevRelayClient.js

```diff
@@ -33,7 +39,11 @@ class DevRelayClient {
     // Start by registering in the relayer hub
     const txParams = payload.params[0];
     const hub = await createRelayHubFromRecipient(this.web3, txParams.to);
-    await this.register(hub);
+    const isNotRegistered = !(await this.isRegistered(hub))
+    if(isNotRegistered) {
```

Comment:
Nit, but I'd just do the `await` inside the `if` expression to avoid creating unnecessary variables

Reply:
Just wanted to make it more clear ;)